docs/docs/getting-started.server.mdx

```diff
@@ -636,6 +636,33 @@ well.
 
 See [Using MDX with Next.js][next-mdx] for more details.
 
+#### Astro
+
+[Astro](https://astro.build/) is built on [islands architecture](https://docs.astro.build/concepts/islands/),
+allowing you to use React, Vue, Svelte, and more within your MDX files.
+
+After [creating your first Astro project](https://astro.new/), you can [add the MDX integration](https://docs.astro.build/guides/integrations-guide/mdx/):
+
+```shell
+astro add mdx
+# Or use "npx" on Windows
+npx astro add mdx
+```
+
+And (optionally) add your favorite component framework(s).
+You can import Markdown, Astro, and other MDX files as components
+out-of-the-box.
+
+```shell
+astro add preact
+# Or use "npx" on Windows
+npx astro add preact
+```
+
+[See the Astro documention](https://docs.astro.build/guides/integrations-guide/mdx/)
+for configuring layouts,
+built-in syntax highlighting with Shiki, adding YAML frontmatter, and more.
```

Comment:
```suggestion
To learn how to configure layouts, support YAML frontmatter, and set up Astro’s
syntax highlighting, [see Astro’s guide on their MDX
integration](https://docs.astro.build/guides/integrations-guide/mdx/).
```

I prefer putting the “question” to the user first, answer second.
This *should* (not sure if this is scientifically proven) also improve the situation for people who skim: if they’re like “nope I don’t need that” they can stop reading this sentence and move on.